src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -13080,7 +13010,6 @@ namespace SurfaceGeometry {
     {
         // J. Glazer - March 2017
 
-        Real64 tol = 0.0127; //  1.27 cm = 1/2 inch
```

Comment:
Per discussion with @JasonGlazer this group of functions are using `OneCentimeter` instead of 1.27cm